tests/pytest/fixtures/workloads.py

```diff
@@ -23,20 +23,18 @@ def within_two_miles_of_mit(record):
         "E 5th St",
     ]
     try:
-        if any([street.lower() in record.address.lower() for street in FAR_AWAY_ADDRS]):
-            return False
-        return True
+        return not any([street.lower() in record.address.lower() for street in far_away_addrs])
     except Exception:
         return False
 
 
 def in_price_range(record):
     try:
         price = record.price
-        if type(price) == str:
+        if type(price) is str:
             price = price.strip()
             price = int(price.replace("$", "").replace(",", ""))
-        return 6e5 < price and price <= 2e6
+        return price > 6e5 and price <= 2e6
```

Comment:
Similar question here re: asking the linter to allow `6e5 < price and price <= 2e6`

Reply:
Fixed as with above